lib/hammer_cli_foreman/compute_resource/utils.rb

```diff
@@ -8,6 +8,21 @@ def self.get_image_uuid(compute_resource_id, image_id)
       )
     end
 
+    def self.get_hostgroup_compute_resource_id(hostgroup_id)
+      hostgroup = HammerCLIForeman.record_to_common_format(
+          HammerCLIForeman.foreman_resource(:hostgroups).call(:show, 'id' => hostgroup_id)
+      )
+      compute_resource_id = hostgroup['compute_resource_id']
+      unless compute_resource_id
+
+        compute_resources = HammerCLIForeman.record_to_common_format(
+            HammerCLIForeman.foreman_resource(:compute_resources).call(:index)
```

Comment:
should we add a search for the name instead of getting the entire list (which might be paginated) and looking for the id in it?